typescript/packages/webpack-example/package.json

```diff
@@ -16,6 +17,6 @@
         "webpack-cli": "^3.3.7"
     },
     "dependencies": {
-        "@0x/mesh-browser": "./../../browser/"
+        "@0x/mesh-browser": "^8.2.0"
```

Comment:
We were using relative imports here for a specific reason. It let's us test whether changes we are making would break the example. Is there a reason you want to change it to `^8.2.0`?

Reply:
Good point. Now that we are working in a yarn workspace, this needs to be `file:../browser` instead of the previous `../../browser` so that the relative path resolves in both the root package and the `@0x/mesh-webpack` package.